rcsb/utils/dictionary/DictMethodAssemblyHelper.py

```diff
@@ -213,10 +213,18 @@ def buildContainerAssemblyIds(self, dataContainer, catName, **kwargs):
             #
             tObj = dataContainer.getObj("pdbx_struct_assembly")
             assemblyIdL = tObj.getAttributeValueList("id")
+            #
+            compModelId = None
+            if tObj.hasAttribute("rcsb_comp_model_id"):
+                compModelId = tObj.getValue("rcsb_comp_model_id", 0)
```

Comment:
`_pdbx_struct_assembly.rcsb_comp_model_id` is not defined in the RCSB extension dictionary. Do you want this to be added?

Reply:
The attribute `rcsb_comp_model_id` is tied to the `entry` category. I don't think it needs to be defined in the dictionary since it's not actually included in the schema. Rather, it's a temporary holder for our internal identifier, which is then assigned to the `rcsb_id` data item of the schema.

Reply:
The variable name `tObj` is used in lines 210 and 214 for `entry` and `pdbx_struct_assembly` objects. Is this intentional?

Reply:
Oh whoops, I see what you mean, thanks for pointing that out. Will fix.